testing/percona-xtrabackup/APKBUILD

```diff
@@ -0,0 +1,54 @@
+# Contributor: Peter Szalatnay <theotherland@gmail.com>
+# Maintainer: Peter Szalatnay <theotherland@gmail.com>
+pkgname=percona-xtrabackup
+pkgver=2.4.6
+pkgrel=0
+pkgdesc='Percona XtraBackup is a MySQL non-blocking backup solution for InnoDB and XtraDB data'
+arch=all
+url='http://www.percona.com/software/percona-xtrabackup/'
+license='GPL'
+makedepends='bash bison cmake ncurses-dev libaio-dev libgcrypt-dev zlib-dev curl-dev libev-dev vim'
+source="https://github.com/percona/percona-xtrabackup/archive/percona-xtrabackup-$pkgver.tar.gz
+        fix-posix_timers.patch
+        "
+
+subpackages="$pkgname-doc $pkgname-test:mytest"
+
+builddir="$srcdir/percona-xtrabackup-percona-xtrabackup-$pkgver"
+
+build() {
+        cd "$builddir"
+
+        CFLAGS="$CFLAGS -DSIGEV_THREAD_ID=4"
+
+        cmake . -DBUILD_CONFIG=xtrabackup_release \
+                -DCMAKE_INSTALL_PREFIX=/usr \
+                -DWITH_MAN_PAGES=OFF \
+                -DDOWNLOAD_BOOST=1 \
+                -DWITH_BOOST="$builddir"/libboost \
+                || return 1
```

Comment:
`|| return 1` are deprecated with latest abuild as the functions are now executed with `set -e`. Please remove them all.